packages/csv-parser-orders/test/line-item-state.spec.js

```diff
@@ -5,7 +5,7 @@ import StreamTest from 'streamtest'
 import LineItemStateParser from '../src/parsers/line-item-state'
 
 describe('LineItemStateParser', () => {
-  describe('::constructor', () => {
+  describe('constructor', () => {
     it('should initialize default values', () => {
       const parser = new LineItemStateParser()
```

Comment:
You could wrap all the 
```js
const parse = new LineItemStateParser()
```
 in a `beforeEach` block

Reply:
Updated c9145bf